README.md

```diff
@@ -1 +1,54 @@
-# module1-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# 21st International Exhibition
+# Food, Ingredients and Food Technologies
+
+> Exhibiting at UzFood is a unique opportunity to present your products to a wide audience of professionals, find new partners and customers among representatives of retail chains and wholesale businesses.
+
+![app-screenshot](./images/project.png)
```

Comment:
- [ ] [OPTIONAL]: It would be more descriptive if you added a screenshot of your project, instead of an image used in your project. You should consider changing that!

Reply:
Changed to snapshot from main page